### PR TITLE
rpm: use nftables as default for f41

### PR DESCRIPTION
**Pull request by Luap99**
Use nftables as default driver for fedora 41.

https://fedoraproject.org/wiki/Changes/NetavarkNftablesDefault

### CODE REVIEW

**Comment by Luap99** [comment]
@lsm5 @mheon PTAL

**Comment by mheon** [comment]
LGTM but someone more familiar with RPM should review this

**Comment by lsm5** [comment]
change LGTM. I'd like to manually verify as well with the copr build.

@Luap99 @mheon what's a definitive check I can use for this?

**Comment by Luap99** [comment]
And if you follow https://fedoraproject.org/wiki/Changes/NetavarkNftablesDefault#How_To_Test then do not create the containers.conf file as this change should switch the default when no config file is set.

**Review by lsm5 — APPROVED** [review]
tried rawhide rpm from packit copr job:
```
$ /usr/libexec/podman/netavark version
{
  "version": "1.12.0-dev",
  "commit": "83c59ef719633a6c45fbb6d3436a1136b0ffb0a8",
  "build_time": "2024-07-30T00:00:00+00:00",
  "target": "x86_64-unknown-linux-gnu",
  "default_fw_driver": "nftables"
}
```

/lgtm
/approve
/hold

**Comment by lsm5** [comment]
Container networking seems fine too. 

Also, PTAL if it matters: 

```
$ sudo nft list table inet netavark
table inet netavark {
	chain INPUT {
		type filter hook input priority filter; policy accept;
		ip saddr 10.88.0.0/16 udp dport 53 accept
	}

	chain FORWARD {
		type filter hook forward priority filter; policy accept;
		ct state invalid drop
		jump NETAVARK-ISOLATION-1
		ip daddr 10.88.0.0/16 ct state established,related accept
		ip saddr 10.88.0.0/16 accept
	}

	chain POSTROUTING {
		type nat hook postrouting priority srcnat; policy accept;
		meta mark & 0x00002000 == 0x00002000 masquerade
		ip saddr 10.88.0.0/16 jump nv_2f259bab_10_88_0_0_nm16
	}

	chain PREROUTING {
		type nat hook prerouting priority dstnat; policy accept;
		fib daddr type local jump NETAVARK-HOSTPORT-DNAT
	}

	chain OUTPUT {
		type nat hook output priority dstnat; policy accept;
		fib daddr type local jump NETAVARK-HOSTPORT-DNAT
	}

	chain NETAVARK-HOSTPORT-DNAT {
	}

	chain NETAVARK-HOSTPORT-SETMARK {
		meta mark set meta mark | 0x00002000
	}

	chain NETAVARK-ISOLATION-1 {
	}

	chain NETAVARK-ISOLATION-2 {
	}

	chain NETAVARK-ISOLATION-3 {
		oifname "podman0" drop
		jump NETAVARK-ISOLATION-2
	}

	chain nv_2f259bab_10_88_0_0_nm16 {
		ip daddr 10.88.0.0/16 accept
		ip daddr != 224.0.0.0/4 masquerade
	}
}
```

**Comment by Luap99** [comment]
looks good

**Pull request merged**
